ui/src/components/policy/AddRuleForm.js

```diff
@@ -122,14 +124,20 @@ export default class AddRuleForm extends React.Component {
     }
 
     inputChanged(key, evt) {
-        this.setState({ [key]: evt.target.value });
-        this.props.onChange(key, evt.target.value);
+        if (key == 'case') {
+            this.setState({ [key]: !this.state.case });
```

Comment:
evt.target.checked

Reply:
Why use it? I'll still need to check the key

Reply:
Fixed